tools/ci_build/amd_hipify.py

```diff
@@ -205,7 +193,9 @@ def hipify(src_file_path, dst_file_path):
     if not os.path.exists(dir_name):
         os.makedirs(dir_name, exist_ok=True)
     # Run hipify-perl first, capture output
-    s = subprocess.run([get_hipify_path(), src_file_path], stdout=subprocess.PIPE, universal_newlines=True).stdout
+    s = subprocess.run(
+        [get_hipify_path(), "-roc", src_file_path], stdout=subprocess.PIPE, universal_newlines=True, check=False
```

Comment:
@xinyazhang or @ytaous,  I wonder what is the reason to add -roc here? Thanks!

Reply:
I found this: https://github.com/ROCm-Developer-Tools/HIPIFY/issues/504